resources/views/component/travelletter.blade.php

```diff
@@ -0,0 +1,27 @@
+<div class="c-travel-letter">
```

Comment:
What is travel-letter? Blog? Lets keep it consistent and rename blog to something better in one go.